### PR TITLE
[DOC]: improve local deployment documentation

### PR DESCRIPTION
**Pull request by Bam92**
## What
Improve the local deployment doc following Google style

## Recommended reading order
N/A

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [ ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

<details><summary><strong>Connector Generator</strong></summary>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed

</details>

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

*Put your acceptance tests output here.*

</details>

### CODE REVIEW

**Comment by marcosmarxm** [comment]
Can you explain
> Improve the local deployment doc following Google style

Airbyte uses docsaurus to build the documentation.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
See my comments, you're removing a lot of info and break the integration with Docsaurus.

File: docs/deploying-airbyte/local-deployment.md [hunk 1]
```diff
@@ -1,65 +1,39 @@
 # Local Deployment
+This page guides you through setting up your Airbyte on your local machine. 
 
-:::info
-These instructions have been tested on MacOS, Windows 10 and Ubuntu 20.04.
+## Prerequisites
+* [Install Docker](https://docs.docker.com/get-docker/) and make sure you have the latest version of `docker-compose` installed.
+* Additionally, for Windows, install [WSL 2 backend](https://docs.docker.com/desktop/windows/wsl/).
```
Comment (Amruta-Ranade) [inline docs/deploying-airbyte/local-deployment.md:6]: Change "Additionally, for Windows, install [WSL 2 backend](https://docs.docker.com/desktop/windows/wsl/)." to "Additionally, for Windows, install [WSL 2 backend](https://docs.docker.com/desktop/windows/wsl/) and enable Hyper-V Windows Features"

File: docs/deploying-airbyte/local-deployment.md [hunk 2]
```diff
@@ -1,65 +1,39 @@
...
 
-:::
+> **_Note:_** These instructions have been tested on MacOS, Windows 10 and Ubuntu 20.04.
```
Comment (Amruta-Ranade) [inline docs/deploying-airbyte/local-deployment.md:8]: Add comma (,) after Windows 10

File: docs/deploying-airbyte/local-deployment.md [hunk 3]
```diff
@@ -1,65 +1,39 @@
...
 
-## Setup & launch Airbyte
+## Deploy on Mac and Linux
+To set up Airbyte on your local machine:
 
-- Install Docker on your workstation \(see [instructions](https://www.docker.com/products/docker-desktop)\). Make sure you're on the latest version of `docker-compose`.
-- After Docker is installed, you can immediately get started locally by running:
+1. Clone the Airbyte repository.
+2.  to the cloned folder (`airbyte`) and start the server with `docker compose`
```
Comment (Amruta-Ranade) [inline docs/deploying-airbyte/local-deployment.md:14]: Missing word: Move (it should be "Move into the cloned folder...")
Add semicolon (:) at the end of the sentence

File: docs/deploying-airbyte/local-deployment.md [hunk 4]
```diff
@@ -1,65 +1,39 @@
...
-BASIC_AUTH_USERNAME=your_new_username_here
-BASIC_AUTH_PASSWORD=your_new_password_here
-```
-
-- Start moving some data!
+3. In your browser, visit http://localhost:8000
+> **_Note:_** You will be asked for a username and password. The default username is `airbyte` and the password is `password`.
 
 ## Deploy on Windows
+To install on Windows:
```
Comment (Amruta-Ranade) [inline docs/deploying-airbyte/local-deployment.md:25]: Change to "To set up Airbyte on your Windows machine:"

File: docs/deploying-airbyte/local-deployment.md [hunk 5]
```diff
@@ -1,65 +1,39 @@
...
-
-Make sure to select the options:
-
-1. _Enable Hyper-V Windows Features_
-2. _Install required Windows components for WSL 2_\
-   when prompted. After installation, it will require to reboot your computer.
-
-**3. You're done!**
+1. Clone the Airbyte repository.
+2.  to the cloned folder (`airbyte`) and start the server with `docker compose`
```
Comment (Amruta-Ranade) [inline docs/deploying-airbyte/local-deployment.md:28]: Same comments as above

**Review by Amruta-Ranade — CHANGES_REQUESTED** [review]

**Review by Amruta-Ranade — APPROVED** [review]

File: docs/deploying-airbyte/local-deployment.md [hunk 6]
```diff
@@ -1,65 +1,39 @@
...
-- Install Docker on your workstation \(see [instructions](https://www.docker.com/products/docker-desktop)\). Make sure you're on the latest version of `docker-compose`.
-- After Docker is installed, you can immediately get started locally by running:
+1. Clone the Airbyte repository.
+2. Move into the cloned folder (`airbyte`) and start the server with `docker compose`;
 
 ```bash
 git clone https://github.com/airbytehq/airbyte.git
 cd airbyte
-docker-compose up
+sudo docker compose up
```
Comment (marcosmarxm) [inline docs/deploying-airbyte/local-deployment.md:19]: why the `sudo`? This is not necessary if you have docker added to your permission group.

File: docs/deploying-airbyte/local-deployment.md [hunk 7]
```diff
@@ -1,65 +1,39 @@
...
-```yaml
-# Proxy Configuration
-# Set to empty values, e.g. "" to disable basic auth
-BASIC_AUTH_USERNAME=your_new_username_here
-BASIC_AUTH_PASSWORD=your_new_password_here
-```
-
-- Start moving some data!
+3. In your browser, visit http://localhost:8000
+> **_Note:_** You will be asked for a username and password. The default username is `airbyte` and the password is `password`.
```
Comment (marcosmarxm) [inline docs/deploying-airbyte/local-deployment.md:22]: @Amruta-Ranade the syntax **_Note_** is native markdown, why not use docsaurus?

File: docs/deploying-airbyte/local-deployment.md [hunk 8]
```diff
@@ -1,65 +1,39 @@
...
-
-**2. Install Docker Desktop on Windows.**
-
-Install [Docker Desktop](https://docs.docker.com/desktop/windows/install/) from here.
-
-Make sure to select the options:
-
-1. _Enable Hyper-V Windows Features_
-2. _Install required Windows components for WSL 2_\
-   when prompted. After installation, it will require to reboot your computer.
```
Comment (marcosmarxm) [inline docs/deploying-airbyte/local-deployment.md:49]: This information is important why are you removing it?

File: docs/deploying-airbyte/local-deployment.md [hunk 9]
```diff
@@ -1,65 +1,39 @@
...
-
-- Start moving some data!
+3. In your browser, visit http://localhost:8000
+> **_Note:_** You will be asked for a username and password. The default username is `airbyte` and the password is `password`.
 
 ## Deploy on Windows
+To set up Airbyte on your Windows machine:
 
-After installing the WSL 2 backend and Docker you should be able to run containers using Windows PowerShell. Additionally, as we note frequently, you will need `docker-compose` to build Airbyte from source. The suggested guide already installs `docker-compose` on Windows.
-
```
Comment (marcosmarxm) [inline docs/deploying-airbyte/local-deployment.md:34]: This is really relevant, why are you removing?

File: docs/deploying-airbyte/local-deployment.md [hunk 10]
```diff
@@ -1,65 +1,39 @@
...
+2. Move into the cloned folder (`airbyte`) and start the server with `docker compose`
 
 ```bash
 git clone https://github.com/airbytehq/airbyte.git
 cd airbyte
-docker-compose up
+sudo docker compose up
 ```
+3. In your browser, visit http://localhost:8000
+4. You will be asked for a username and password. The default username is `airbyte` and the password is `password`.
```
Comment (marcosmarxm) [inline docs/deploying-airbyte/local-deployment.md:36]: I like the note, but would be nice to link to a document tutorial explaining how to change the default one.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
I'm not confident if removing some information from this page is good to users. 
The proposal page is basically a duplicate between MacOS/Linux and Windows.

File: docs/deploying-airbyte/local-deployment.md (hunk 7 above)
Comment (Amruta-Ranade) [reply docs/deploying-airbyte/local-deployment.md:22]: Agreed. I'll fix the formatting issues for all deployment docs PRs in a follow-up PR

File: docs/deploying-airbyte/local-deployment.md (hunk 6 above)
Comment (Amruta-Ranade) [reply docs/deploying-airbyte/local-deployment.md:19]: @Bam92 Please fix this

File: docs/deploying-airbyte/local-deployment.md (hunk 10 above)
Comment (Amruta-Ranade) [reply docs/deploying-airbyte/local-deployment.md:36]: I don't think we have one yet. I'll add a link when we do :)

File: docs/deploying-airbyte/local-deployment.md (hunk 8 above)
Comment (Amruta-Ranade) [reply docs/deploying-airbyte/local-deployment.md:49]: Added as a prerequisite

**Comment by marcosmarxm** [comment]
Hello :wave:, first thank you for this amazing contribution. 

We really appreciate the effort you've made to improve the project.
We ask you patience for the code review. Last month our team was focused on Hacktoberfest event and that probably left some PR without the proper feedback. And this week, due to the Thanksgiving US Holiday, most our team is out of office with their families. Another important piece of information why code won't be merge this week is: as a safety measure the core team has decided to freeze merging code to main branch to keep the release stable. Next week we'll return to you with the proper code review and update the status of your contribution.

If you have any questions feel free to send me a message in Slack!
Thanks!

**Comment by marcosmarxm** [comment]
@Amruta-Ranade @Bam92 this is updated?

**Comment by Amruta-Ranade** [comment]
> @Amruta-Ranade @Bam92 this is updated?

No, I'll handle this in 2023

**Comment by Amruta-Ranade** [comment]
> @Amruta-Ranade can we close or merge this?

We can close it.

**Pull request closed without merging**